tox.ini

```
@@ -1,7 +1,8 @@
 [tox]
 envlist =
     py{36,37,38,39,py3}-dj{22,30,31,32}-{unittest,pytest}
-    py{38,39}-djmain-{unittest,pytest}
+    py{38,39,310}-djmain-{unittest,pytest}
+    py{310}-dj{32,40}-{unittest,pytest}
```

4.0 is also supported by 3.8,3.9 so maybe 🤔

```suggestion
    py{38,39,310}-dj{40,main}-{unittest,pytest}
    py310-dj32-{unittest,pytest}
```